src/components/Settings/SettingsSection.tsx

```diff
@@ -1,13 +1,13 @@
-import { Box, Flex } from '@radix-ui/themes'
+import { Flex } from '@radix-ui/themes'
 
 type SettingsSectionProps = {
   children: React.ReactNode
 }
 
 export function SettingsSection({ children }: SettingsSectionProps) {
   return (
-    <Flex gap="2" direction="column" pt="4" p="1">
-      <Box>{children}</Box>
+    <Flex direction="column" pt="4" p="1" height="100%">
+      {children}
```

Comment:
I made some changes to the parent elements in the tab content so we use flexbox more efficiently with `children` items.